python/lsst/pipe/analysis/utils.py

```diff
@@ -1028,7 +1037,23 @@ def calibrateSourceCatalogPhotoCalib(dataRef, catalog, fluxKeys=None, zp=27.0):
         catalog[fluxKey] /= factor
```

Comment:
I'm still very sad about the use of `factor`. I thought it was used only in the case of `photoCalib.instFluxToStuff` not working due to lack of instflux errors, but now I see you **always** divide by factor?! Is this actually necessary? I don't think we want everything on an arbitrary 27th mag flux zero point without making it very clear that is what we are doing. Can we take it out or make the after-the-fact zero point adjustment optional (not default)?

Reply:
I think this is out of scope of the current ticket.